actionpack/lib/action_dispatch/http/response.rb

```diff
@@ -403,9 +402,14 @@ def cookies
     ContentTypeHeader = Struct.new :mime_type, :charset
     NullContentTypeHeader = ContentTypeHeader.new nil, nil
 
-    def parse_content_type
-      content_type = get_header CONTENT_TYPE
+    def parse_content_type(content_type = get_header(CONTENT_TYPE))
       if content_type
+        # Mime::Type objects are sometimes passed as arguments, so we need to
+        # call #to_s on them, to be able to call #split a few lines later.
```

Comment:
Why are `Mime::Type` objects *sometimes* passed as arguments?  We should always be getting a string.